hydra-node/json-schemas/api.yaml

```diff
@@ -269,27 +269,27 @@ components:
           timestamp:
             $ref: "#/components/schemas/UTCTime"
 
-    ReadyToCommit:
-      title: ReadyToCommit
+    HeadIsInitializing:
+      title: HeadIsInitializing
       description: |
-        A `Init` transaction has been observed on-chain by the given party who's now ready to commit into the initialized head.
+        A `Init` transaction has been observed on-chain, with the given HeadId
       payload:
         type: object
         required:
           - tag
+          - headId
           - parties
-          - seq
```

Comment:
All server outputs should have a sequence number.. why would this not have one?

Must: fix the api.yaml to include `seq` as this one will also have a `seq` field because of `ToJSON TimedServerOutput`